docs/source/contrib_docs/getting-started.rst

```diff
@@ -1,22 +1,14 @@
-=================================
```

Comment:
The changes were to both improve consistency of naming of headings and to aid in discoverability of docs. I'm happy to change new in the name; I was simply making a distinction from the old migrated from IPython dev guide.